query-string/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): { [key: string]: string | string[] };
+    export function parse(str: string): { [key: string]: string | string[] | null };
```

Comment:
Yes, for `stringify` `| undefined` must be specified for beeing strictNullChecks safe.

This PR focuses solely on `parse`.